frontend/src/scenes/session-recordings/apm/playerInspector/ItemPerformanceEvent.tsx

```diff
@@ -232,110 +273,79 @@ export function ItemPerformanceEvent({
                     <>
                         <StatusRow item={item} />
                         <p>
-                            Request
-                            <StartedAt item={item} /> took <b>{humanFriendlyMilliseconds(item.duration)}</b>
-                            {sizeInfo.formattedDecodedBodySize || sizeInfo.formattedBytes ? (
-                                <>
-                                    {' '}
-                                    to load <b>{sizeInfo.formattedDecodedBodySize || sizeInfo.formattedBytes}</b> of
-                                    data
-                                </>
-                            ) : null}
-                            {sizeInfo.isFromLocalCache ? (
-                                <>
-                                    {' '}
-                                    <span className="text-muted">(from local cache)</span>
-                                </>
-                            ) : null}
-                            {sizeInfo.formattedCompressionPercentage && sizeInfo.formattedEncodedBodySize ? (
-                                <>
-                                    , compressed to <b>{sizeInfo.formattedEncodedBodySize}</b> saving{' '}
-                                    <b>{sizeInfo.formattedCompressionPercentage}</b>
-                                </>
-                            ) : null}
-                            .
+                            Request <StartedAt item={item} /> <DurationDescription item={item} />
+                            <SizeDescription sizeInfo={sizeInfo} />.
                         </p>
                     </>
                     <LemonDivider dashed />
-                    {['fetch', 'xmlhttprequest'].includes(item.initiator_type || '') ? (
```

Comment:
was also confusing to hide _all_ the tabs in this case instead of not showing some tabs

Reply:
if I'd seen the JSON earlier, I'd have fixed this earlier 🙈